src/index.js

```diff
@@ -49,19 +49,22 @@ function parse(model, response, options = {}) {
 export default Bookshelf => {
   const Model = Bookshelf.Model.prototype;
   const client = Bookshelf.knex.client.config.client;
-  const parseOnFetch = client === 'sqlite' || client === 'sqlite3';
+  const handleOnSaving = client !== 'mysql';
```

Comment:
The thing is that I would like to reproduce the error, adding a test where a query fails without using the plugin and another fixing it with the plugin as it's our convention. 

Anyway if I can't do that I'll remove that constraint and turn on the parsing on saving across all clients, although tests on MySQL will get weird.

I currently don't have time to investigate further, can you please use the `hotfix/mysql-support` in the meantime?